astropy/cosmology/io/html.py

```diff
@@ -0,0 +1,161 @@
+import astropy.cosmology.units as cu
+import astropy.units as u
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.cosmology.parameter import Parameter
+from astropy.table import QTable
+
+from .table import from_table, to_table
+
+# Format look-up for conversion, {original_name: new_name}
+# TODO! move this information into the Parameters themselves

```

Comment:
Definitely for follow-up, but seconding this TODO - could follow what is done for units perhaps?

Reply:
Yeah. I have a PR to start on this, but Parameters are a bit different than units because they are descriptors, so can't be used with `__format__`

Reply:
Ah, yes, so something like a `to_string` method is indeed not possible (at least not directly).

Reply:
But the information can be stored on the Parameter instance, and then ``to_string`` would be a class method and ``Parameter.to_string(Cosmology.<parameter>, cosmo.<parameter>)`` would work. I've been playing around with this in https://github.com/astropy/astropy/pull/12983.